pack/tic/itd_encounter.json

```diff
@@ -1,19 +1,531 @@
+		"text": "<b>Forced</b> - After this location is revealed: Spawn the set-aside Innsmouth Shoggoth at this location, exhausted.\n[action]: <b>Parley.</b> Move 1 barrier between this location and a connecting location so it is between this location and a different connecting location.\n[free] If the investigators control 2 or more keys, as a group: <b>Parley.</b> Read <b>Flashback V</b> in the Campaign Guide.",
+		"traits": "Innsmouth. Midtown.",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_flavor": "Finding a trustworthy individual to query may open a door, and an out-of-town guest staying at the hotel could provide a vital piece of this puzzle.",
+		"clues": 2,
+		"code": "07140",
+		"double_sided": true,
+		"encounter_code": "in_too_deep",
+		"encounter_position": 18,
+		"faction_code": "mythos",
+		"illustrator": "Marc Simonetti",
+		"name": "Gilman House",
+		"pack_code": "itd",
+		"position": 140,
+		"quantity": 1,
+		"shroud": 3,
+		"text": "<b>Forced</b> - After this location is revealed: Randomly choose 1 of the set-aside facedown keys and place it on this location without looking at it.\n[action]: <b>Parley.</b> Test [combat] (2) or [agility] (2). If you succeed, remove 1 barrier between this location and a connecting location.",
+		"traits": "Innsmouth. Midtown.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"Crossing the bridge and turning toward the roar of the lower falls, I passed close to the Marsh refinery, which seemed oddly free from the noise of industry. This building stood on the steep river bluff near a bridge and an open confluence of streets which I took to be the earliest civic centre...\"\n - H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"clues": 1,
+		"clues_fixed": true,
+		"code": "07141",
+		"double_sided": true,
+		"encounter_code": "in_too_deep",
+		"encounter_position": 19,
+		"faction_code": "mythos",
+		"illustrator": "Mark Molnar",
+		"name": "Marsh Refinery",
+		"pack_code": "itd",
+		"position": 141,
+		"quantity": 1,
+		"shroud": 3,
+		"text": "[free] If you control 1 or more keys: Move to a connecting location.\n[action] Discard an asset you control: <b>Parley.</b> Remove 1 barrier between this location and a connecting location.",
+		"traits": "Innsmouth. Coastal. Midtown.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"No railroad now - B. & M. never went through, and the branch line from Rowley was given up years ago.\"\n - H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"clues": 0,
+		"code": "07142",
+		"double_sided": true,
+		"encounter_code": "in_too_deep",
+		"encounter_position": 20,
+		"faction_code": "mythos",
+		"illustrator": "Alexander Chelyshev",
+		"name": "Railroad Station",
+		"pack_code": "itd",
+		"position": 142,
+		"quantity": 1,
+		"shroud": 1,
+		"text": "<b>Forced</b> - After you enter this location, if it is flooded: Test [agility] (2). If you fail, take 1 damage.\n[action]: <b>Resign</b>. You escape the submerged town.",
```

Comment:
Please can you move the period inside the bold tags for resign.: **Resign.** not **Resign**.